rdb/storage_test.go

```diff
@@ -592,8 +592,8 @@ func TestStorage_GetBestTrial(t *testing.T) {
 		t.Errorf("error: %v != nil", err)
 		return
 	}
-	if bestTrial.Value == 0.2 && bestTrial.Number != 3 {
-		t.Errorf("want Trial(Value=0.2, Number: 3), but got %#v", bestTrial)
+	if bestTrial.Value == 0.2 && bestTrial.Number != 2 {
+		t.Errorf("want Trial(Value=0.2, Number: 2), but got %#v", bestTrial)
```

Comment:
The number starts from 1 on the current implementation. But Optuna's number starts from 0.